examples/creatives/nonlinear_overlay.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * A sample SIMID nonlinear ad that shows an overlay of
+ * buttons to showcase functionality
+ */
+class NonLinear extends BaseSimidCreative {
+    constructor() {
+      super();
+
+      this.addButtonClickActions_();
+    }
+
+    /**
+     * Adds actions to different buttons available on the overlay.
+     * @private
+     */
+    addButtonClickActions_() {
+      this.sendMessageOnButtonClick_("close_ad", CreativeMessage.REQUEST_STOP);
+    }
+
+    onRequestResize() {
+      const resize_x_val = document.getElementById('resize_x_val').value;
+      const resize_y_val = document.getElementById('resize_y_val').value;
+      const resize_width = document.getElementById('resize_width').value;
+      const resize_height = document.getElementById('resize_height').value;
+  
+      const resize_params = {
```

Comment:
I think you made a bunch of changes to this in a separate PR.  Please use those updated changes.